docs: Remove tool name parameters from docs as they were removed (#3086)
n8n-io/n8n-docs: docs/integrations/builtin/cluster-nodes/sub-nodes/n8n-nodes-langchain.toolvectorstore.md

## Node parameters

### Description of Data

Enter a description of the data in the vector store.

> n8n uses the **Node Name** and **Description of Data** parameters to populate the tool description for AI agents using the following format:

Reviewer: ```suggestion
n8n uses the node name (select the name to edit) and **Description of Data** parameter to populate the tool description for AI agents using the following format:
```

> > Useful for when you need to answer questions about [Node Name]. Whenever you need information about [Description of Data], you should ALWAYS use this. Input should be a fully formed question.

Reviewer: ```suggestion
> Useful for when you need to answer questions about [node name]. Whenever you need information about [Description of Data], you should ALWAYS use this. Input should be a fully formed question.
```

> Spaces in **Node Name** are converted to underscores on the tool description, and special characters are not allowed.

Reviewer: ```suggestion
Spaces in the node name are converted to underscores in the tool description. Special characters aren't allowed.
```

Are special characters removed, do they break something, or does the field not accept them?

Author: The name field accepts the usual characters our nodes normally accept, but like on other tools you are unfortunately able to insert some special characters on the node names that langchain doesn't support.

Commonly spaces are something our users might enter on the tool name, and they are converted to underscores, so if user names the node "Test Name" its passed as `Test_Name` to the tool description that the agent sees.

If user names the tool with some other special characters they'll see an error when the agent runs.

Reviewer: Okay cool. I was just wondering about what specifically happened when you attempted to add "not allowed" characters. I'll update it slightly then to mention that it'll cause problems for the agent. Thanks for clarifying!

Author: <img width="1265" alt="image" src="https://github.com/user-attachments/assets/e8e4489b-46ea-4a92-b5ce-6f9b620fdbf7" />

This is what happens if you have tools with names that contain characters that langchain doesn't support. Not optimal, and I suppose this is why the name field was once added on these couple of nodes, but they were removed now in favor of having every tool work the same.

> Spaces in the node name are converted to underscores in the tool description. Special characters aren't allowed.

Reviewer: ```suggestion
Spaces in the node name are converted to underscores in the tool description. Special characters aren't allowed and will result in errors when the agent runs.
```
